README.md

```diff
@@ -83,6 +83,11 @@ This library is intended to provide a quicker and easier way to get started usin
 * `uint8_t getAddress(void)`<br>
   Returns the current I&sup2;C address.
 
+* `bool begin(void)`<br>
+  Iniitializes the sensor to use Wire as the default port. 
```

Comment:
Typo "Initializes" How about "Initializes the two wire interface using the default two wire interface, Wire."